lib/models/movie.js

```diff
@@ -28,11 +42,18 @@ module.exports = Bookshelf.Model.extend({
   },
 
   serialize: function () {
-    return {
-      id: this.get('id'),
-      title: this.get('name'),
-      release_year: this.get('release_year'),
-      object: 'movie'
-    };
+    return this.locations()
+    .then((locations) => {
+      const serializedLocations = locations.map((location) => {
+        return location.serialize();
+      });
```

Comment:
nit: if you wanted to you could inline this and skip the `return`. We don't really have any lint convention on line length, so it wouldn't be an issue from that standpoint.

```js
locations.map((location) => location.serialize());
```